android/src/main/java/com/christopherdro/htmltopdf/RNHTMLtoPDFModule.java

```diff
@@ -59,15 +47,6 @@ public void convert(final ReadableMap options, final Promise promise) {
         fileName = UUID.randomUUID().toString();
       }
 
-      if (options.hasKey("fonts")) {
-        if (options.getArray("fonts") != null) {
-          final ReadableArray fonts = options.getArray("fonts");
-          for (int i = 0; i < fonts.size(); i++) {
-            customFonts.add(fonts.getString(i));
-          }
-        }
-      }
-
```

Comment:
Do we loose support for custom fonts?

Reply:
I believe we don't. Simply any HTML & CSS combination can be supported by a web browser will be supported by this solution.

Reply:
https://stackoverflow.com/questions/3900658/how-to-change-font-face-of-webview-in-android

I think not-preinstalled fonts need to be placed at `android/app/src/main/assets` and can be used with css:

```html
<style type="text/css">
@font-face {
    font-family: MyFont;
    src: url("file:///android_asset/fonts/MyFont.otf")
}
body {
    font-family: MyFont;
    font-size: medium;
    text-align: justify;
}
</style>
```

Didn't found a solution fitting current API.

Reply:
Oh my bad! It was pretty late in Germany, sorry... As custom font are a Android only feature and the fonts should be already in this directory (right?) we could just inject the above css to the supplied html to match the current API.